src/lib/ddcci.c

```diff
@@ -361,6 +362,8 @@ static int i2c_read(struct monitor* mon, unsigned int addr, unsigned char *buf,
 		struct i2c_msg             i2cmsg;
 		int i;
 	
+		memset(&msg_rdwr, 0, sizeof(msg_rdwr));
```

Comment:
I'd just do:
```
struct i2c_rdwr_ioctl_data msg_rdwr = { 0 };
```

Reply:
Does `... = { 0 }` set whole struct to zero?

Reply:
Yep (https://stackoverflow.com/questions/11152160/initializing-a-struct-to-0?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa has a good reference too C99 standard).